src/app/temperatureConverter/temperatureConverter.component.ts

```diff
@@ -10,7 +10,32 @@ export class TemperatureConverter implements OnInit {
 
   constructor() {}
 
+  celsius: number = null;
+  fahrenheit: number = null;
+
   ngOnInit() {}
 
+  conversion(from: string) {
```

Comment:
Good job on the challenge, I would modify this with a more reusable name, and would pass types to be more easy to do the "onchange" function in the input.

something like: 

```
 onChange(value: string | null, type: "c" | "f")  
```